qa/integration-tests/src/test/java/io/camunda/it/migration/util/TasklistUtil.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.it.migration.util;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonParser;
+import io.camunda.webapps.schema.entities.tasklist.TaskEntity.TaskImplementation;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.awaitility.Awaitility;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.Network;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
+
+public class TasklistUtil {
```

Comment:
🔧 Can we maybe call it differently? Util sounds so... general. Like TasklistMigrationHelper, TasklistMigrationContainer or TasklistContainerSupport or something

🔧 We could even think about split this to Tasklistclient and TasklistContainer.
Maybe we can also make this an extension even. I think would be useful for other tests as well like: StandaloneCamundaTest (id did something similar, for Operate, see OperateClientTest to access v1)

Reply:
I guess we can iterate over it 👍🏼 the last point is nothing we need to do it immediately but good to have in mind.